hw/ip/otbn/dv/otbnsim/sim/model.py

```diff
@@ -0,0 +1,367 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+from random import getrandbits
+from typing import List, Optional, Tuple, cast
+
+from attrdict import AttrDict  # type: ignore
+
+from riscvmodel.model import Model, State, Environment  # type: ignore
+from riscvmodel.isa import Instruction, TerminateException  # type: ignore
```

Comment:
```suggestion
from riscvmodel.model import Model, State, Environment, TerminateException  # type: ignore
from riscvmodel.isa import Instruction  # type: ignore
```

Reply:
Yeah, I'll do that (and bump the minimum version of riscvmodel accordingly). On a point release...